### PR TITLE
Improve PR instructions

### PR DESCRIPTION
**Pull request by NovemberTang**
Thanks to @michaelwmcnamara for his invaluable feedback on the PR body

## What does this change?

While testing this feature on some repos belonging to other DevX streams, we received feedback that the instructions were not as clear as they could be. This PR aims to rectify that, by using more specific language, and clarifying terms where required.

We now also log the PR title and body on CODE and DEV

## Why?

To make it easier for teams to follow the instructions

## How has it been verified?

- PR body still formats as expected
- Verified that the Python checklist item shows up when Python is listed as a language, and vice versa.

### CODE REVIEW

File: packages/snyk-integrator/src/snyk-integrator/snyk-integrator.ts [hunk 1]
```diff
@@ -84,6 +84,31 @@ function checklist(items: string[]): string {
...
+	const step2 =
+		'The Snyk job should run automatically on every commit to this branch. ' +
+		'Click through on the Snyk status check see the logs of the latest run on ' +
+		'this PR, and verify it has generated one project per dependency manifest ' +
+		'(except pnpm and deno). ' +
+		'Examples of dependency manifests are a build.sbt, or a package-lock.json, ' +
+		'essentially, any file that lists the dependencies of your project.';
+	const step3 =
+		`When you are happy the action works, remove the branch name \`${branchName}\`` +
+		'trigger from the snyk.yml (aka delete line 6), approve, and merge.';
```
Comment (akash1810) [inline packages/snyk-integrator/src/snyk-integrator/snyk-integrator.ts:104]: Interested to hear opinions on https://github.com/guardian/service-catalogue/pull/712, which tries to create a simpler approach to creating long, multi-line strings.

Comment (NovemberTang) [reply packages/snyk-integrator/src/snyk-integrator/snyk-integrator.ts:104]: Looks useful, though would need modifying to suit this use case. These are transformed into checklist items, so each step is a single line.

**Review by tjsilver — APPROVED** [review]
👍

File: packages/snyk-integrator/src/snyk-integrator/snyk-integrator.ts (hunk 1 above)
Comment (akash1810) [reply packages/snyk-integrator/src/snyk-integrator/snyk-integrator.ts:104]: Using the `stripMargin` function means string concatenation goes away:

```ts
const step1 = stripMargin`
  |Replace the SNYK_ORG variable with the org name that your team already uses (you should have other repos integrated with Snyk.
  |If you can’t find any, reach out to DevX).
  |Examples are 'guardian-devtools' and 'guardian-dotcom-n2y'`;
```

**Pull request merged**